micrometer-jakarta9/src/test/java/io/micrometer/jakarta9/instrument/mail/DefaultMailSendObservationConventionTests.java

```diff
@@ -221,26 +207,19 @@ void subjectShouldBeThereWhenSet() throws MessagingException {
     }
 
     @Test
-    void protocolAddressAndPortShouldBeThereWhenSet() {
```

Comment:
I dropped this as it seems to be a duplicate of the `shouldHaveLowCardinalityKeyValues()`.